src/connectors.js

```diff
@@ -143,7 +143,7 @@ export function getConnectors(chainId, connectorsInitsOrConfigs = {}) {
       },
       handleActivationError(err) {
         if (err instanceof WalletConnectUserRejectedRequestError) {
-          throw new RejectedActivationError()
+          return new ConnectionRejectedError()
```

Comment:
Just wanted to sanity check here... are these supposed to be `return`s?

Reply:
Yes; we don't want to throw the error again, as we're in a `catch` statement already. What we wanna do is get the error, and set it to state it's available on the `error` prop.